src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -137,12 +142,24 @@ class ConfigureIntegration extends AsyncView<Props, State> {
           </Form>
         )}
 
-        {integration.dynamicDisplayInformation?.configure_integration?.instructions.map(
-          (instruction, i) => (
-            <Alert type="info" key={i}>
-              <span dangerouslySetInnerHTML={{__html: singleLineRenderer(instruction)}} />
-            </Alert>
-          )
+        {instructions && instructions.length > 0 && (
+          <Alert type="info">
+            {instructions?.length === 1 ? (
+              <span
+                dangerouslySetInnerHTML={{__html: singleLineRenderer(instructions[0])}}
+              />
+            ) : (
+              <List symbol={<IconArrow size="xs" direction="right" />}>
+                {instructions?.map((instruction, i) => (
+                  <ListItem key={i}>
+                    <span
+                      dangerouslySetInnerHTML={{__html: singleLineRenderer(instruction)}}
+                    />
+                  </ListItem>
+                )) ?? []}
```

Comment:
I believe you can also remove this

Reply:
I need this for typescript reasons unfortunately